compat/src/index.js

```diff
@@ -470,6 +440,10 @@ options.vnode = vnode => {
 		setSafeDescriptor(type.prototype, 'componentWillUpdate');
 		type._patchedLifecycles = true;
 	}
+
+	// TODO: Removing this saves 16 bytes!
+	vnode.preactCompatNormalized = true;
```

Comment:
Anyone know if we need to support this property? Or can we remove it?

Reply:
I vote for removing it, it's unused and don't really see a use case for it

Reply:
Removed. Since this comment is now outdated (which I think means it'll be collapsed for some), I made a new comment for others to see and discuss.